Framework/PythonInterface/mantid/utils.py

```diff
@@ -0,0 +1,70 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from contextlib import contextmanager
+from importlib import import_module
+
+
+def import_mantid(modulename, package="", caller_globals=None):
```

Comment:
Could you help me understand the need for the `caller_globals`?

Reply:
Good spot, I'll add it to the docstring, but it is because previously we did e.g. `from _kernel import *`, which would insert all of `_kernel`'s dict into the caller namespace. 

`import_module`, `__import__(..., fromlist['*'])` do not do that, they simply give the module back as an object, and I wasn't able to find a way to replicate the `from X import *` functionality.

The workarounds I found were to either do `exec('from X import *') in caller_globals` or this `caller_globals.update(lib.__dict__)` which seems better.

Reply:
Ah I see yes. I forgot we don't need the `*` import in `mantidqt` so I was wondering about the difference. This is fine.